zone/lua_parser_events.cpp

```diff
@@ -210,18 +210,30 @@ void handle_npc_death(QuestInterface *parse, lua_State* L, NPC* npc, Mob *init,
 
 void handle_npc_cast(QuestInterface *parse, lua_State* L, NPC* npc, Mob *init, std::string data, uint32 extra_data,
 						  std::vector<std::any> *extra_pointers) {
-	int spell_id = std::stoi(data);
-	if(IsValidSpell(spell_id)) {
-		Lua_Spell l_spell(&spells[spell_id]);
-		luabind::adl::object l_spell_o = luabind::adl::object(L, l_spell);
-		l_spell_o.push(L);
-		lua_setfield(L, -2, "spell");
-	} else {
-		Lua_Spell l_spell(nullptr);
-		luabind::adl::object l_spell_o = luabind::adl::object(L, l_spell);
-		l_spell_o.push(L);
-		lua_setfield(L, -2, "spell");
-	}
+
+        Seperator sep(data.c_str());
```

Comment:
Need to use tabs for indent on all this instead of spaces

Reply:
Agreed.